changelogs/fragments/266-default-database-for-mysql-user.yml

```
@@ -0,0 +1,2 @@
+minor_change:
```

```suggestion
minor_changes:
```
this should fix sanity tests

---

*facepalm* thanks, i was wondering whats wrong...

---

you're welcome!